components/Cases/CaseLink.spec.tsx

```diff
@@ -54,4 +54,22 @@ describe('CaseLink component', () => {
     const { asFragment } = render(<CaseLink {...props} />);
     expect(asFragment()).toMatchInlineSnapshot(`<DocumentFragment />`);
   });
+
+  it('should render properly - with handled link for viewing warning notes', () => {
+    const props = {
+      recordId: '123',
+      caseFormData: mockedWarningNote.caseFormData,
+    };
+    const { asFragment } = render(<CaseLink {...props} />);
+    expect(asFragment()).toMatchInlineSnapshot(`
```

Comment:
I might use `.toMatchSnapshot()` here instead of `toMatchInlineSnapshot()`. It (sort of) made sense to use an inline snapshot for the previous test, because it was empty, but now you have a proper snapshot we show push the snapshot over to a file

Reply:
Done in 1eeb478